### PR TITLE
Added spa@1.0.7

### PR DESCRIPTION
**Pull request by zhaoda**

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Seems there is no `dist` under latest `spa` npm package.

File: ajax/libs/spa/package.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  "npmName": "spa",
+  "npmFileMap": [
+    {
+      "basePath": "/dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "spa.js"
```
Comment (PeterDaveHello) [inline ajax/libs/spa/package.json]: And should point to minified file.

**Comment by zhaoda** [comment]
fixed

**Comment by PeterDaveHello** [comment]
Please squash them into only one commit, thanks.

**Pull request closed without merging**